Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN npm run disclaim
 
 FROM nginx:1.17-alpine
 RUN mkdir -p /var/www/mender-gui/dist
+RUN apk update && apk add curl
```

Comment:
to prevent the `apk` cache from filling the image you might want to use `RUN apk add --no-cache curl` or use the included `wget`

Reply:
sure fixed.